2019/07.md

```diff
@@ -94,18 +94,18 @@ Supporting materials includes slides, a link to the proposal repository, a link
     | ✓ | 3     | 15m     | [`Promise.allSettled`](https://github.com/tc39/proposal-promise-allSettled) for Stage 4 ([slides](https://docs.google.com/presentation/d/1qhYDRlsvyVUtveT2tS0mJtzG-xe9islO6-t2wMQUJso/edit)) | Mathias Bynens |
     |  | 2     | 30m     | [`String.prototype.replaceAll`](https://github.com/tc39/proposal-string-replace-all) for Stage 3 ([slides](https://docs.google.com/presentation/d/194gQ-GRfb9r17Vva9nevePkFUfPck1o_pXHM25LlVKs/edit)) | Mathias Bynens |
     |  | 2     | 30m     | ~~(Withdraw from discussion- champion like to keep in stage 2 due to last minutes concern/suggestion need more time to addresss)   [`Intl.DisplayNames`](https://github.com/tc39/proposal-intl-displaynames) for Stage 3 ([slides](http://shorturl.at/yPSZ1))~~ | Frank Tang  |
-    |  | 2     | 30m     | [Collection Normalization update](https://docs.google.com/presentation/d/1xxkHqtScIvdCBI4IZOpWHh7AKCJs6s5edQQu06wnZYc/edit) | Bradley Farias |
-    | ✓ | 2     | 30m     | [RegExp Match Offsets](https://github.com/tc39/proposal-regexp-match-offsets) Update / Stage 3 ([slides](https://1drv.ms/p/s!AjgWTO11Fk-TkdY-MYw52uP-lJjreA)) | Ron Buckton |
-    | ✓ | 2     | 30m     | Update on [function implementation hiding](https://github.com/tc39/proposal-function-implementation-hiding) ([slides](https://docs.google.com/presentation/d/1lWH97DxTLU3_1EJA-F19uIzagZQx7PZmys7WyNXw3cY/edit?usp=sharing)) | Michael Ficarra |
-    | ✓ | 2     | 45m     | [Nullish Coalescing](https://github.com/tc39/proposal-nullish-coalescing/) for Stage 3 ([slides](https://1drv.ms/p/s!AltPy8G9ZDJdqSUtMZeOKLg1RcRD)) | Daniel Rosenwasser |
+    |  | 2     | 30m     | ~~[Collection Normalization update](https://docs.google.com/presentation/d/1xxkHqtScIvdCBI4IZOpWHh7AKCJs6s5edQQu06wnZYc/edit)~~ | Bradley Farias |
```

Comment:
We don't cross them out anymore (per @ecmageneva's request). The first column is the "done" column.

Reply:
oh okay thanks